ui/components/src/DateTimeRangePicker/AbsoluteTimePicker.tsx

```diff
@@ -124,13 +164,12 @@ export const AbsoluteTimePicker = ({ initialTimeRange, onChange }: AbsoluteTimeF
               minDateTime={timeRange.start}
               onChange={(newValue) => {
                 if (newValue === null) return;
-                setTimeRange((current) => {
-                  return { start: current.start, end: newValue };
-                });
+                onChangeEndTime(newValue);
               }}
-              onAccept={() => {
```

Comment:
I found a really subtle bug in here where you would lose the minutes value for the end time because of a race condition where `onChange` for the minute selection and `onAccept` would trigger at about the same time.

Reply:
Good catch, can't believe I didn't notice this before!